cmd/controller/main.go

```diff
@@ -66,11 +73,26 @@ func setupChecks(mgr ctrl.Manager) error {
 }
 
 func runController(watchNamespace string, imageServer imagehandler.ImageHandler, envInputs *env.EnvInputs) error {
+	infraEnvRequirement, err := labels.NewRequirement(infraEnvLabel, selection.DoesNotExist, nil)
+	if err != nil {
+		setupLog.Error(err, "cannot create an infraenv label filter")
+		return err
+	}
+
+	cacheOptions := cache.Options{
+		SelectorsByObject: cache.SelectorsByObject{
```

Comment:
This appears to be saying that only resources with the label should be reconciled. I thought we wanted the opposite? Or maybe I don't understand how the cache configuration works?

Reply:
The DoesNotExist is the key, but it cannot be inline and the naming is confusing. I'd suggest renaming `infraEnvRequirement` to `excludeInfraEnv`.